mantidproject/fitbenchmarking
# Added Test Coverage Tracking

#### Description of Work

Fixes #112 
Added test coverage tracking with coveralls. Now using a testing framework named pytest to get coverage data and export it to coveralls.

#### Testing Instructions

Go to https://coveralls.io/github/mantidproject/fitbenchmarking to check the coverage.
